test/unit/tracks/text-track-settings.test.js

```diff
@@ -9,6 +9,16 @@ const tracks = [{
   label: 'test'
 }];
 
+const defaultSettings = {
```

Comment:
We're *almost* in agreement ;-) 

[x] The dropdowns shouldn't have any defaults or "empty values" 
[ ] See YouTube and iOS - the "checkbox" (whether for each value or for the settings as a whole) defines whether _styling in the WebVTT overrides the values set by the user_. That's not quite the same as whether or not to change what vtt.js does - the vtt.js defaults would disappear, and be replaced by the settings in video.js's Settings Dialog, although those setting values may or may not be changed by settings in the WebVTT file.

I can see the value in being able to completely turn off the Captions Settings functionality and not changing what vtt.js does (as was discussed elsewhere), and that could be implemented as a separate PR with a video.js parameter to control it.

Implementing this behavior of Captions Settings control won't be easy with the existing vtt.js though, and it's interface to `Services` and `WebVTTSet` object. We would need to fork vtt.js, and hope we can push a better user control API upstream to them.